### ISSUE TITLE
Validator reports error on later line with data from earlier line

### ISSUE THREAD

**Issue by robredpath** [spilio/openreferral-playground#8]
I tried out the validator with quite a large file that I prepared for @greggish a while back for the Miami pilot. I got an error on row 1042 that there was an invalid format for the URL (which there is for ~75% of the 3k-line file), but the URL reported was the one from line 1. 

I wasn't able to replicate the behaviour with a constructed test file, however, so not sure how much of an issue this is. 

@greggish - what's the license on the Miami data? Can I share it here?

**Comment by greggish** [comment]
Yes you can share here. It's out of date! We don't want anyone using it for real-purposes. But yeah.

**Comment by robredpath** [comment]
@spilio The data I was using was the services.csv file from https://drive.google.com/open?id=0B3YdI6zzzaUXQTkyYU5GNHFEamM
